core/src/main/scala/chisel3/StrongEnum.scala

```diff
@@ -409,3 +409,33 @@ private[chisel3] class UnsafeEnum(override val width: Width) extends EnumType(Un
   override def cloneType: this.type = new UnsafeEnum(width).asInstanceOf[this.type]
 }
 private object UnsafeEnum extends EnumFactory
+
+/** Suppress enum cast warnings
+  *
+  * Users should use [[EnumFactory.safe <EnumType>.safe]] when possible.
+  *
+  * This is primarily used for casting from [[UInt]] to a Bundle type that contains an Enum.
+  * {{{
+  * class MyBundle extends Bundle {
+  *   val addr = UInt(8.W)
+  *   val op = OpEnum()
+  * }
+  *
+  * // Since this is a cast to a Bundle, cannot use OpCode.safe                                                                                    
```

Comment:
```suggestion
  * // Since this is a cast to a Bundle, cannot use OpCode.safe
```
This is probably my fault in copy-paste but there's a bunch of whitespace on the end of this line